03.asynchronous/callback.js

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+
+import sqlite3Module from "sqlite3";
+import timers from "timers/promises";
+const sqlite3 = sqlite3Module.verbose();
+const db = new sqlite3.Database(":memory:");
+
+db.run(
+  "CREATE TABLE IF NOT EXISTS books(id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL UNIQUE)",
+  function () {
+    db.run(
+      "INSERT INTO books(title) VALUES (?)",
```

Comment:
スタイルが整ってないです。以下のいずれかにしてください。

```suggestion
      "INSERT INTO books (title) VALUES (?)",
```

```suggestion
      "INSERT INTO books(title) VALUES(?)",
```